sciencer/policies.py

```diff
@@ -9,6 +9,7 @@ class Policy(Enum):
 
     BY_DOI = "By DOI"
     BY_AUTHOR = "By Author"
+    BY_QUERY = "By Query"
```

Comment:
The policies are used to match providers capabilities with the requesters - i.e. Collectors, Filters, Expanders - needs. Maybe it makes sense to call it capabilities rather than policies.

Reply:
Regarding the query policy while creating implementing `get_paper_by_terms`, I name them this way because the provider needs to have the entry point to be searched by query, however the collector uses the concept of terms.

Would you suggest moving the abstraction to a higher level such that the provider as a method called `get_paper_by_query` and the collector is responsible for converting the list of terms to a query format?

Reply:
Not necessarily.

I think renaming Policies to Capabilities would be more suitable.
For the next pull request ;)